lib/src/components/NxCopyToClipboard/types.ts

```diff
@@ -19,9 +18,9 @@ export interface Props extends Omit<NxFormGroupProps, 'children' | 'isRequired'>
   inputProps?: ExternalInputProps | null;
 }
 
-export const propTypes: ValidationMap<Props> = {
+export const propTypes: PropTypes.ValidationMap<Props> = {
   ...omit(['children', 'isRequired'], nxFormGroupPropTypes),
   content: PropTypes.string.isRequired,
   onCopyUsingBtn: PropTypes.func,
-  inputProps: PropTypes.shape(omit(['type', 'value', 'isPristine', 'readOnly'], nxTextInputPropTypes))
+  inputProps: PropTypes.shape(omit(['type', 'value', 'isPristine'], nxTextInputPropTypes))
```

Comment:
QQ: why do we remove this prop?

Reply:
> QQ: why do we remove this prop?

I don't really remember removing that, but presumably it was because `nxTextInputPropTypes` doesn't have a `readOnly` field to omit in the first place.  It might by that the newer versions of typescript and ramda's props caught that at compile-time when they previously didn't.